web-ng/src/app/services/data-import/data-import.service.ts

```diff
@@ -35,11 +35,13 @@ export class DataImportService {
     projectId: string,
     layerId: string,
     file: File
-  ): Observable<ImportCsvResponse> {
+  ): Promise<ImportCsvResponse> {
```

Comment:
We should remove unused `Observable` import statement from line 20?

Reply:
Good catch!